browser-extension/background/background.js

```
@@ -0,0 +1,451 @@
+  }
+}
+
+// Initialize API instance
+const mediaFusionAPI = new MediaFusionAPI();
+
+// Message handling for communication with content scripts and popup
+const handleMessage = async (message, sender, sendResponse) => {
+  try {
+    switch (message.action) {
+      case 'getSettings':
+        const settings = await mediaFusionAPI.getSettings();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)